.travis.yml

```diff
@@ -7,6 +7,9 @@ os: osx
 osx_image: xcode6.4
 
 env:
+  matrix:
+    
+    - CONDA_PY=27
```

Comment:
Now it is working! But I had to use my hacked version of conda-smithy to find it...